en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md

```diff
@@ -353,11 +353,11 @@ Follow the steps below to create the task and schedule it.
     ```xml
     <task class="org.apache.synapse.startup.tasks.MessageInjector" group="synapse.simple.quartz" name="PrintStockQuoteScheduledTask" xmlns="http://ws.apache.org/ns/synapse">
                 <trigger count="1" interval="5" />
-                <property name="to" value="http://localhost:9000/soap/SimpleStockQuoteService" xmlns:task="http://www.wso2.org/products/wso2commons/tasks" />
+                <property name="to" value= http://localhost:9000/soap/SimpleStockQuoteService />
```

Comment:
Don't we need quotation marks here?

Reply:
Fixed [1aa1beb](https://github.com/wso2/docs-apim/pull/7282/commits/1aa1beb0e2b8ffe48563aded26dcb628c8b4030a)